src/calendar/header/index.tsx

```diff
@@ -227,14 +230,20 @@ const CalendarHeader = forwardRef((props: CalendarHeaderProps, ref) => {
     const shouldDisable = isLeft ? disableArrowLeft : disableArrowRight;
     const onPress = !shouldDisable ? isLeft ? onPressLeft : onPressRight : undefined;
     const imageSource = isLeft ? require('../img/previous.png') : require('../img/next.png');
-    const renderArrowDirection = isLeft ? 'left' : 'right';
-
+    const renderArrowDirection = isLeft ? 'left' : 'right';   
+    const hitSlop: Insets | undefined = useMemo(() =>
```

Comment:
React Hook "React.useMemo" cannot be called inside a callback.
React Hooks must be called in a React function component or a custom React Hook function. (react-hooks/rules-of-hooks).
Please move the `useMemo` (the `hitSlop` const) outside the `_renderArrow` callback.

Reply:
Moved hitSlop outside the `_renderArrow` callback.